k8s/chart/templates/rbac.yaml

```diff
@@ -0,0 +1,56 @@
+{{ if .Values.rbac.create}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "sdp-operator.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "sdp-operator.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "1"
```

Comment:
I don't think this RBAC needs to be a hook.

I had to add these annotations for sdp-k8s-client because the preinstall hook was running `kubectl` commands which required these roles - I needed to make sure the RBAC are created before the 'meat' of the chart gets created.

Reply:
That should not be there. Leftover